.github/workflows/tf-apply.yml

```diff
@@ -0,0 +1,103 @@
+name: Terraform-and-test
+run-name: run by ${{ github.actor }}
+on:
+    - push
+    - workflow_dispatch
+
+
+jobs:
+  plan:
+    name: 'Run Terraform Plan'
+    runs-on: ubuntu-22.04
+    permissions:
+      issues: write
+    
+    defaults:
+      run:
+        working-directory: ${{ vars.TERRAFORM_WORKING_DIR }}
+
+    steps:
+        - name: 'Checkout'
+          uses: actions/checkout@v3.5.2
+
+        - name: HashiCorp - Setup Terraform
+          uses: hashicorp/setup-terraform@v2.0.3
+          with:
+            cli_config_credentials_hostname: app.terraform.io
+            terraform_version: latest
+            terraform_wrapper: true
+
+        - name: Configure AWS Credentials For GitHub Actions
+          uses: aws-actions/configure-aws-credentials@v2
+          with:
+            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-region: ${{ vars.AWS_REGION }}
+
+        - name: Create Artifact Folder 
+          shell: bash
+          run: |
+            sudo mkdir -p -m777 ${{ github.workspace }}/tfplanoutput
```

Comment:
Sorry, I missed that in the previous review. Does 777 is necessary here?

Reply:
"others" must have rwx for terraform to be able to access the directory. Not ideal, but I changed it to 707